### ISSUE TITLE
[Feature]:  Introduce Ntex web framework Service

### ISSUE THREAD

**Issue by pavlospt** [shuttle-hq/shuttle#1708]
### Describe the feature

Hello, I would like to add Ntex us an option in Shuttle. I will open a PR for it and just raising this issue for awareness and contributing completeness. Will follow with Clerk example for Ntex as well :) 

### Duplicate declaration

- [X] I have searched the issues and this feature has not been requested before.

**Comment by jonaro00** [comment]
Thanks!
Feel free to make an examples PR for a hello-world example.
For the clerk example, do you envision it being a copy of the actix-web+clerk example, or something new?

**Comment by pavlospt** [comment]
Yes most probably it will be a copy with updated references since Ntex is like the successor of Actix!

**Comment by jonaro00** [comment]
The clerk example uses an actix middleware in clerk-rs, so it might not be as simple as copy+pasting...
Moreover, having multiple variants of the same example can be a bit burdensome, especially the fullstack ones, so feel free to come up with something new that none of the other examples are using. Maybe there's some cool feature in ntex worth showing off and building an API around?

**Comment by pavlospt** [comment]
TBH Ntex and Actix are pretty similar so the same middleware structure can be built around it, but I will check it out if I am able to solve the runtime panic issue and come up with something new!

**Comment by sourabpramanik** [comment]
> TBH Ntex and Actix are pretty similar so the same middleware structure can be built around it, but I will check it out if I am able to solve the runtime panic issue and come up with something new!

Yes, we can create a middleware around it. For this example, it might be out of scope but I am willing to contribute to the middleware of clerk-rs lib for Ntex, and then you may use that.  What do you think?
